[rospy.client][INFO] 2021-01-16 07:40:30,825: init_node, name[/train_636_1610782830794], pid[636]
[xmlrpc][INFO] 2021-01-16 07:40:30,826: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-01-16 07:40:30,827: Started XML-RPC server [http://192.168.0.191:45959/]
[rospy.init][INFO] 2021-01-16 07:40:30,828: ROS Slave URI: [http://192.168.0.191:45959/]
[rospy.impl.masterslave][INFO] 2021-01-16 07:40:30,828: _ready: http://192.168.0.191:45959/
[rospy.registration][INFO] 2021-01-16 07:40:30,830: Registering with master node http://192.168.0.126:11311
[xmlrpc][INFO] 2021-01-16 07:40:30,830: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2021-01-16 07:40:30,928: registered with master
[rospy.rosout][INFO] 2021-01-16 07:40:30,928: initializing /rosout core topic
[rospy.rosout][INFO] 2021-01-16 07:40:30,936: connected to core topic /rosout
[rospy.simtime][INFO] 2021-01-16 07:40:30,939: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2021-01-16 07:40:30,955: topic[/bev/temporal_bev_image] adding connection to [http://192.168.0.126:36903/], count 0
[rospy.internal][INFO] 2021-01-16 07:40:30,959: topic[/odom] adding connection to [http://192.168.0.126:45431/], count 0
[rospy.internal][INFO] 2021-01-16 07:40:30,965: topic[/start_goal_points] adding connection to [http://192.168.0.126:40949/], count 0
[rospy.internal][INFO] 2021-01-16 07:40:30,971: topic[/scan] adding connection to [http://192.168.0.126:36669/], count 0
[rospy.internal][INFO] 2021-01-16 07:40:30,973: topic[/scan] adding connection to [http://192.168.0.126:45431/], count 1
[rospy.internal][INFO] 2021-01-16 07:40:30,975: topic[/is_start_episode] adding connection to [http://192.168.0.126:34007/], count 0
[rospy.internal][INFO] 2021-01-16 07:40:31,140: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2021-01-16 07:40:31,144: topic[/episode_watcher/is_finish_episode] adding connection to [/episode_manager], count 0
[rospy.internal][INFO] 2021-01-16 07:40:31,245: topic[/cmd_vel] adding connection to [/gazebo], count 0
[rospy.core][INFO] 2021-01-16 07:40:33,442: signal_shutdown [atexit]
[rospy.internal][INFO] 2021-01-16 07:40:33,447: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2021-01-16 07:40:33,447: topic[/cmd_vel] removing connection to /gazebo
[rospy.internal][INFO] 2021-01-16 07:40:33,448: topic[/episode_watcher/is_finish_episode] removing connection to /episode_manager
[rospy.internal][INFO] 2021-01-16 07:40:33,448: topic[/bev/temporal_bev_image] removing connection to http://192.168.0.126:36903/
[rospy.internal][INFO] 2021-01-16 07:40:33,449: topic[/odom] removing connection to http://192.168.0.126:45431/
[rospy.internal][INFO] 2021-01-16 07:40:33,449: topic[/start_goal_points] removing connection to http://192.168.0.126:40949/
[rospy.internal][INFO] 2021-01-16 07:40:33,449: topic[/scan] removing connection to http://192.168.0.126:36669/
[rospy.internal][INFO] 2021-01-16 07:40:33,450: topic[/scan] removing connection to http://192.168.0.126:45431/
[rospy.internal][WARNING] 2021-01-16 07:40:33,451: Unknown error initiating TCP/IP socket to 192.168.0.126:59449 (http://192.168.0.126:36903/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.internal][INFO] 2021-01-16 07:40:33,451: topic[/is_start_episode] removing connection to http://192.168.0.126:34007/
[rospy.internal][WARNING] 2021-01-16 07:40:33,452: Unknown error initiating TCP/IP socket to 192.168.0.126:35677 (http://192.168.0.126:45431/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.impl.masterslave][INFO] 2021-01-16 07:40:33,452: atexit
